query-advisor/where-to-find.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: 'Where to Find Query Advisor'
+backlink_href: /docs/query-advisor
+backlink_title: 'Query Advisor'
+---
+
+Once Query Advisor is configured and collecting EXPLAIN plans, optimization insights appear in several locations throughout pganalyze:
+
+## Query Advisor Landing Page
+
+Access the dedicated Query Advisor page to view all detected optimization opportunities across your databases, organized by potential impact.
+
+![Query Advisor landing page showing automated EXPLAIN plan analysis with detected optimization opportunities](query-advisor-landing.png)
+
+As pganalyze collects EXPLAIN plans through auto_explain, Query Advisor automatically analyzes them and flags problematic patterns in near real-time.
```

Comment:
I think we usually say near real-time for the checks that are running per compact snapshots (aka every 10 seconds), but this check runs daily.

<img width="356" height="76" alt="Image" src="https://github.com/user-attachments/assets/4778f495-cb26-46d6-8aec-8627b8bafca5" />

Reply:
To confirm, for now I think its better to call it daily, since that's what happens to be visible to the user. The fact that we do near-realtime analysis as a first phase is probably not worth sharing in the docs.